docs/source/using_the_ve/virtual_ecosystem_in_use.md

```diff
@@ -129,12 +92,48 @@ The Virtual Ecosystem writes out a number of data files:
 These files are written to the standard NetCDF data file format.
 
 ```{code-cell} ipython3
+---
+editable: true
+slideshow:
+  slide_type: ''
+tags: []
+---
+# Dependencies for the data and graphing
+import pathlib
+
+import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
+import numpy as np
+import xarray
+```
+
+```{code-cell} ipython3
+---
+editable: true
+slideshow:
+  slide_type: ''
+tags: []
+---
 # Load the generated data files
 initial_state = xarray.load_dataset("/tmp/ve_example/out/initial_state.nc")
```

Comment:
If a user install the develop branch using `pip` they probably won't have the initial state file by default settings, see https://github.com/ImperialCollegeLondon/virtual_ecosystem/issues/544 I can confirm that this is still the case today on my computer.